pkg/manifest/print.go

```diff
@@ -17,15 +17,21 @@ package manifest
 import (
 	"fmt"
 	"io"
+	"net/http"
 	"sort"
 	"strings"
 	"text/tabwriter"
 
+	"github.com/golang/protobuf/jsonpb"
 	"github.com/tetratelabs/getenvoy-package/api"
 )
 
-// PrettyPrint tabwrites the passed manifest to the passed writer
-func PrettyPrint(writer io.Writer, manifest *api.Manifest) error {
+// Print retrieves the manifest from the passed URL and writes it to the passed writer
+func Print(writer io.Writer, manifestURL string) error {
+	manifest, err := fetch(manifestURL)
```

Comment:
Hmm, I don't think fetching should be in the print package and the Print func... It seems not relevant to "print"s. What is the downside of the fetch package?

Reply:
As a consumer of the manifest API I just want to print the manifest, how to retrieve the manifest is an implementation detail. It could be at a URL, it could be a local file, etc.

To make it clearer, I should rename the argument to just `manifestLocation`. Then, error on Non-URL inputs saying that only URL locations are supported currently.

Reply:
I see, then make sense to me 👍